apps/src/templates/progress/ProgressLessonTeacherInfo.jsx

```diff
@@ -89,7 +89,7 @@ export default connect(state => ({
   hasNoSections: state.teacherSections.sectionsAreLoaded &&
     state.teacherSections.sectionIds.length === 0
 }), dispatch => ({
-  toggleHidden(scriptName, sectionId, lessonId, hidden) {
-    dispatch(toggleHidden(scriptName, sectionId, lessonId, hidden));
+  toggleHiddenStage(scriptName, sectionId, lessonId, hidden) {
+    dispatch(toggleHiddenStage(scriptName, sectionId, lessonId, hidden));
```

Comment:
Is there a reason we're not passing an object for `mapDispatchToProps` here?  It looks like we're not transforming or rebinding any arguments.
```
}), { toggleHiddenStage })(ProgressLessonTeacherInfo);
```

Reply:
I think I didn't know about this pattern when I'd originally done the work (there are a few places in the codebase like this). Will fix it up now